lib/src/react_client/component_registration_utils.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:react/react.dart';
```

Comment:
I would vote for just `component_registration.dart` instead of `component_registration_utils.dart`